dc-cudami-server/dc-cudami-server-webapp/src/main/resources/application-typedeclarations.yml

```diff
@@ -11,9 +11,11 @@ cudami:
       - CONTAINED
       - IMAGES
       - ISSUE
+      - MIKROSKOPAUFNAHMEN
       - NEWSPAPER
       - PART
       - SERIAL
+      - SPEKTRALOPTIMIERTE BILDER
```

Comment:
wäre hier auch `SPEKTRALOPTIMIERTE_BILDER` möglich? So Enum-Style :wink:

Reply:
Ist geändert, und wir bleiben in einer Sprache (Englisch)